Octokit.Reactive/Clients/IObservableGitHubAppsClient.cs

```diff
@@ -53,5 +61,41 @@ public interface IObservableGitHubAppsClient
         /// </remarks>
         /// <param name="installationId">The Id of the GitHub App Installation</param>
         IObservable<AccessToken> CreateInstallationToken(long installationId);
+
+        /// <summary>
+        /// List installations for user
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/apps/#list-installations-for-user</remarks>
+        IObservable<InstallationsResponse> GetAllInstallationsForUser();
+
+        /// <summary>
+        /// List installations for user
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/apps/#list-installations-for-user</remarks>
+        IObservable<InstallationsResponse> GetAllInstallationsForUser(ApiOptions options);
+
+        /// <summary>
+        /// Enables an authenticated GitHub App to find the organizations's installation information.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/apps/#find-repository-installation</remarks>
+        /// <param name="owner">The owner of the repo</param>
+        /// <param name="repo">The name of the repo</param>
+        IObservable<Installation> GetRepositoryInstallation(string owner, string repo);
```

Comment:
I wonder if this should be named `GetRepositoryInstallationForCurrent()` since it uses the currently authenticated GitHub App... or otherwise `FindRepositoryInstallation()`?

Reply:
also we need to support the `repositoryId` version of this call in addition to the `owner/repo` version